src/test/contract.rs

```diff
@@ -30,9 +29,11 @@ fn test_initialize() {
     );
 
     assert_eq!(
-        result.len() as usize,
+        did_document.did.len() as usize,
         "did:chaincerts:".len() + ENCODED_MSI_LEN
     );
+
+    assert_eq!(did_document.context, context);
```

Comment:
We should match other struct fields as well, not only the context.
We could make the `DIDContractTest::setup()` function returns the DID document to make the assertions easier.